src/sink/ws.rs

```diff
@@ -195,17 +394,18 @@ impl offramp::Impl for Ws {
             let (reply_tx, _) = bounded(1);
 
             Ok(SinkManager::new_box(Self {
-                sink_url: TremorURL::from_onramp_id("ws")?, // dummy value
-                codec: Box::new(crate::codec::null::Null {}),
-                preprocessors: vec![],
-                postprocessors: vec![],
-                is_linked: false,
-                connections: HashMap::new(),
+                sink_url: TremorURL::from_onramp_id("ws")?,  // dummy value
```

Comment:
should we populate this with a bit more details?

Reply:
this is overwritten with the actual instance url on `init`, so i think we are good here.

Reply:
👍